tree/tree/src/TBranch.cxx

```diff
@@ -1279,6 +1279,14 @@ Int_t TBranch::GetEntry(Long64_t entry, Int_t getall)
             fNextBasketEntry = -1;
             return -1;
          }
+         if (fTree->GetMaxVirtualSize() < 0) {
```

Comment:
Since we would be using a 'new' bit it should be fine.

We also have to, independently, decide whether this is a persistent or transient property.  (both fBits and a bool datamember) allow both.